### DISCUSSION TITLE
Problem with `cue vet` with regex

### DISCUSSION THREAD

**Discussion by shubhamrai1993** [cue-lang/cue#1961] [Q&A] [answered]
I have a cue file containing the following snippet - 
```cue
{
    arn: =~"^[\w]*$"
}
```

When I run `cue vet` against it, I get this error - 
```
unknown escape sequence:
    ./test.cue:2:16
```

The regex seems correct to me. What is wrong with the regex if indeed that is the issue?

**Comment by slewiskelly** [comment, marked as answer]
You'll need to use either: 

```cue
{
    arn: =~ #"^[\w]+$"#
}
```

or:

```cue
{
    arn: =~ "^[\\w]+$"
}
```

As per the [language spec](https://cuelang.org/docs/references/spec/#string-and-byte-sequence-literals):

> Several escape sequences allow arbitrary values to be encoded as ASCII text. An escape sequence starts with an escape delimiter, which is \ by default. The escape delimiter may be altered to be \ plus a fixed number of hash symbols # by padding the start and end of a string or byte sequence literal with this number of hash symbols.
